examples/kind/test/kind-config.yaml

```diff
@@ -1,5 +1,5 @@
 kind: Cluster
-apiVersion: kind.sigs.k8s.io/v1alpha3
+apiVersion: kind.sigs.k8s.io/v1alpha2
```

Comment:
Why did you change this?

Reply:
I will check because i did locally to use the new kind and the charts installation dod not work

Reply:
I have experienced an issue with kind 0.3.0. Might be due to this. We should skip 0.3.0 and go to 0.4.0 directly.

Reply:
setting to 0.5.1
sorry for the long delay.... :(